packages/web-api/src/WebClient.spec.js

```diff
@@ -817,6 +817,18 @@ describe('WebClient', function () {
         });
     });
 
+    it('should set retrySec info on the response_metadata object', function () {
```

Comment:
I think this test could use a different description that describes the *functionality* it's testing, rather than what the code in the test does 🙂. The current description might be confusing if a new contributor makes a change and sees that this test doesn't pass, only to find that `retrySec` isn't part of the test body 😕.

 It looks like this test checks that some(?) metadata is present on a response. A different description would make this a stellar test 🤩!

Reply:
Awesome, thanks for the feedback. I'm on it!